main.cpp

```diff
@@ -79,42 +99,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+int measureDogSize(int dogLength, int dogHeight, int dogBreadth = 0)
+{
+  ignoreUnused(dogLength, dogHeight, dogBreadth);
+  return{};
+}
 /*
  2)
  */
-
+bool measureDogBuoyancy(char dogSize = 'L', int dogWeight=0, float volumeOfAirInDog = 0.0f)
+{
+  ignoreUnused(dogSize, dogWeight, volumeOfAirInDog);
+  return{};
+}
 /*
  3)
  */
-
+int measureCarDurability(int carMileage = 0, bool carDamage = false)
+{
+  ignoreUnused(carMileage, carDamage);
+  return{};
+}
 /*
  4)
  */
-
+void playNote(int octave = 4, char note = 'C', int length = 1, int projectTempo = 120)
+{
+  ignoreUnused(octave, note, length, projectTempo);
+}
 /*
  5)
  */
-
+int calculateCalories(int distanceCovered, int runningSpeed, int weight = 0)
+{
+  ignoreUnused(distanceCovered, runningSpeed, weight);
+  return{};
+}
 /*
  6)
  */
-
+int trackLocation(int xPath, int yPath)
+{
+  ignoreUnused(xPath, yPath);
+  return{};
+}
 /*
  7)
  */
-
+void throwBalls(int ballDistance, int numberOfBalls=0, bool doBallsExplode = false)
+{
+  ignoreUnused(ballDistance, numberOfBalls, doBallsExplode);
+}
 /*
  8)
  */
-
+void cookSpaghetti(int numberOfNoodles, int durationOfBoiling=0, bool hasSauce = false, bool hasMeatballs = false)
+{
+  ignoreUnused(numberOfNoodles, durationOfBoiling, hasSauce, hasMeatballs);
+}
 /*
  9)
  */
-
-/*
- 10)
- */
+char observeDogFeatures(bool areEarsFloopy = false, char dogColor = 'r', char dogFurPattern = 's')
+{
+  ignoreUnused(areEarsFloopy, dogColor, dogFurPattern);
+  return{};
+}
+/*
+ 10)
+ */
+int makeCupOfCoffee(int amountPowder = 0, int amountWater = 1, int amountSugar = 0)
+{ 
+  ignoreUnused(amountPowder, amountWater, amountSugar);
+  return{};
+}
```

Comment:
Indent this correctly.
Conform to the course coding standard.   it can be found in the readme.md
- [ ] revise